appendages/electronic_component_detector_list.py

```diff
@@ -40,13 +67,13 @@ def get_command_functions(self):
         rv = "void Decode() {\n"
         rv += "\tchar pad = cmdMessenger.readBinArg<char>();\n"
         rv += "\tchar code[5];\n"
-        rv += "\tecd.decode(pad, code, false);\n"
+        rv += "\tint rv = 0;\n"
```

Comment:
That seems like a good idea. We could potentially have non-conflicting names for things between C and Python.